share/models/ingest.py

```diff
@@ -132,10 +132,18 @@ def natural_key(self):
         return (self.label,)
 
     def get_harvester(self, **kwargs):
-        return self.harvester.get_class()(self, **{**kwargs, **(self.harvester_kwargs or {})})
+        """Return a harvester instance configured for this SourceConfig.
+
+        Passed kwargs override or add to harvester_kwargs.
+        """
+        return self.harvester.get_class()(self, **{**(self.harvester_kwargs or {}), **kwargs})
```

Comment:
I think I found a good compromise -- preserve passing kwargs to fetch/harvest methods, but also pass the config's harvester_kwargs into `_do_fetch` the same way (and let passed kwargs override those in the config). It unifies the kwargs, lets all of them be per-call, and gets rid of `self.kwargs`, which made me feel dirty and full of regret.